src/node.cc

```diff
@@ -3366,16 +3365,18 @@ void LoadEnvironment(Environment* env) {
     ReportException(env, try_catch);
     exit(10);
   }
+  // The bootstrap_node.js file returns a function 'f'
   CHECK(f_value->IsFunction());
   Local<Function> f = Local<Function>::Cast(f_value);
 
   // Now we call 'f' with the 'process' variable that we've built up with
-  // all our bindings. Inside node.js we'll take care of assigning things to
-  // their places.
+  // all our bindings. Inside bootstrap_node.js we'll take care of
+  // assigning things to their places.
 
   // We start the process this way in order to be more modular. Developers
-  // who do not like how 'src/node.js' setups the module system but do like
-  // Node's I/O bindings may want to replace 'f' with their own function.
+  // who do not like how 'lib/internal/bootstrap_node.js' setups the module
```

Comment:
Perhaps putting the full path here is unnecessary since the filename itself isn't ambiguous?